Move orgs.yaml to its own folder (#2933)
kubevirt/project-infra: docs/debugging-resources/prow.md

**Note: some of these need to be a symlink to the real files, please follow these and edit in target location.**

| Type         | Holds configuration for                |
|--------------|----------------------------------------|
| config.yaml  | Prow components, prow job presets      |
| plugins.yaml | Prow plugins                           |
| mirror.yaml  | image mirror                           |
| jobs         | Prow job configurations per repository |
| labels.yaml  | GitHub labels[^1]                      |
| orgs.yaml    | GitHub organization settings[^2]       |

> [^1]: GitHub labels for pull requests and issues are updated periodically by cron job [label-sync-kubevirt] and [label-sync-nmstate]

Reviewer: ```suggestion
[^1]: GitHub labels for pull requests and issues are updated periodically by the cron jobs [label-sync-kubevirt] and [label-sync-nmstate].
```
markdown doesn't respect the newline when built so we need a fullstop to distinguish sentences.

  On changes the config is checked by presubmits pull-prow-kubevirt-labels-update-precheck

> [^2]: KubeVirt GitHub organization settings are enforced by [peribolos]

Reviewer: ```suggestion
[^2]: KubeVirt GitHub organization settings are enforced by [peribolos].
```

>   see [KubeVirt GitHub org automation]

Reviewer: ```suggestion
  See [KubeVirt GitHub org automation] for more information. 
```
